perun-auditparser/src/main/java/cz/metacentrum/perun/auditparser/AuditParser.java

```diff
@@ -716,6 +729,10 @@ private static RichResource createRichResource(Map<String, String> beanAttr) {
 		if(beanAttr==null) return null;
 		RichResource richResource = new RichResource();
 		richResource.setId(Integer.valueOf(beanAttr.get("id")));
+		String uuidString = beanAttr.get("uuid");
+		if (uuidString != null && !"null".equals(uuidString)) {
+			richResource.setUuid(UUID.fromString(uuidString));
+		}
```

Comment:
There is missing the change for RichGroup.

Reply:
It is not needed in rich group because it is reusing the method for creating a group where it is present.

Reply:
My bad :)